src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adder = (number) => {
```

Comment:
The `adder` function does not handle non-numeric inputs. According to the task description, you don't need to check if the argument is a number, but you might want to ensure that the function behaves correctly when non-numeric values are passed.